packages/dd-trace/src/proxy.js

```diff
@@ -50,6 +51,7 @@ class Tracer extends NoopProxy {
         }
 
         this._tracer = new DatadogTracer(config)
+        this.appsec = new AppsecSdk(this)
```

Comment:
so do we change this to `this._tracer` ? I'd say yes, as we don't need the noop proxy feature
```suggestion
        this.appsec = new AppsecSdk(this)
```